src/SDKs/Azure.ApplicationModel.Configuration/data-plane/README.md

```diff
@@ -39,3 +39,12 @@ public async Task HelloWorld()
 1. [How to access diagnostic logs](https://github.com/Azure/azure-sdk-for-net/tree/master/src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.Configuration.Tests/samples/Sample4_Logging.cs)
 2. [How to configure retry policy](https://github.com/Azure/azure-sdk-for-net/tree/master/src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.Configuration.Tests/samples/Sample6_ConfiguringRetries.cs)
 3. [How to configure service requests](https://github.com/Azure/azure-sdk-for-net/tree/master/src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.Configuration.Tests/samples/Sample7_ConfiguringPipeline.cs)
+
+
+# Contributing
+If the changes you are working on span both Azure.Base and Azure.Configuration then you can set this environment variable before launching Visual Studio. That will use Project To Project references between Azure.Configuration and Azure.Base instead of package references.
+
+This will enable the project to project references:
+```
+set UseProjectReferenceToAzureBase=true
```

Comment:
While using the env variable maybe OK for now I personally would avoid the env variable and instead have them update the project or a props file setting the property. If we end up with more projects having this pattern I would also consider putting this into a targets file under Azure.Base and have projects that want to reference it import that targets file instead. That way this is controlled in exactly that one place.

Reply:
Sounds good! As discussed offline I'll create a props file in the azure.base project that will be imported by all the projects that depend on it and that controls the logic for package vs project